oracle/Dockerfile

```diff
@@ -70,6 +70,8 @@ RUN \
 # This allows users of this container to choose, whether they want to run the container as sbtuser (non-root) or as root
 USER root
 RUN \
+  mkdir /home/sbtuser/.ivy2 && \
```

Comment:
I guess the other images will have the same problem? Can you also add these commands in there?